src/SPC/builder/unix/library/libjxl.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SPC\builder\unix\library;
+
+use SPC\store\FileSystem;
+use SPC\util\executor\UnixCMakeExecutor;
+use SPC\util\SPCTarget;
+
+trait libjxl
+{
+    public function patchBeforeBuild(): bool
+    {
+        $depsContent = file_get_contents($this->source_dir . '/deps.sh');
+        if (str_contains($depsContent, '# return 0')) {
+            return false;
+        }
+        FileSystem::replaceFileStr(
+            $this->source_dir . '/deps.sh',
+            'return 0',
+            '# return 0',
+        );
+        shell()->cd($this->source_dir)
+            ->exec('./deps.sh');
```

Comment:
Not the biggest fan of this, but I couldn't think of a good solution to do it at the download stage. Maybe a source extract hook?

./deps.sh downloads required other libraries.

Reply:
oh right, thanks